src/app/config.cpp

```diff
@@ -34,6 +34,8 @@ Config::Config()
     this->cam_local_device = this->ia_config.value("Camera/local_device").toString();
     this->cam_is_network = this->ia_config.value("Camera/is_network").toBool();
     this->cam_local_format_override = this->ia_config.value("Camera/local_format_override").value<QVideoFrame::PixelFormat>();
+    this->cam_autoconnect = this->ia_config.value("Camera/automatically_reconnect").toBool();
+    this->cam_autoconnect_time_secs = this->ia_config.value("Camera/auto_reconnect_time_secs", 6).toInt();
```

Comment:
do we even want this to be configurable? I dont see where the user can set it

Reply:
It's not in the user interface. I'm not sure what a good value is and whether or not the right answer will vary from machine to machine so I want people to be able to change it (via the .ini file) if this value is bothersome.